napari/_tests/test_interactive_transforms.py

```diff
@@ -138,3 +138,17 @@ def test_transform_coupling(make_napari_viewer):
 
     viewer.overlays.interaction_box.transform_drag = Affine(scale=[2.0, 2.0])
     np.testing.assert_almost_equal(layer.affine.scale, [0.5, 0.5, 2.0, 2.0])
+
+
+def test_interaction_box_changes_with_layer_transform(make_napari_viewer):
+    viewer = make_napari_viewer()
+    image_layer = viewer.add_image(np.random.random((28, 28)))
+    image_layer.mode = 'transform'
+    initial_selection_box = np.copy(
+        viewer.overlays.interaction_box.transform.affine_matrix
+    )
+    viewer.layers[0].scale = [5, 5]
+    assert not np.allclose(
+        initial_selection_box,
+        viewer.overlays.interaction_box.transform.affine_matrix,
+    )
```

Comment:
Could this test be more specific? Something like:

```
assert np.allclose(box_before * 5, box_after)
```

Reply:
Thanks:) Yes, of course:) I change this, because I need a push to trigger the CI again.